lib/pipe/pipe-windows.c

```diff
@@ -9,6 +9,7 @@
 
 #include <windows.h>
 #include <sddl.h>
+#include "pipe-helpers.h"
```

Comment:
For consistency it would be good to move this include up. We currently try to follow this style:

```
#include "own-header-or-c"

#include "internal-a.h"
#include "internal-b.h"

#include <system-a.h>
#include <system-b.h>
```

So at most three groups separated by a blank line, with includes sorted alphabetically within each group.